mod-data-export/src/test/resources/firebird/dataexport/features/export-for-holdings.feature

```diff
@@ -132,12 +132,13 @@ Feature: Tests export hodings records
     #should return job execution by id and wait until the job status will be 'COMPLETED'
     Given path 'data-export/job-executions'
     And param query = 'id==' + jobExecutionId
-    And retry until response.jobExecutions[0].status == 'COMPLETED' && JSON.stringify(response.jobExecutions[0].progress) == '{"exported":1,"total":1,"failed":0}'
+    And retry until response.jobExecutions[0].status == 'COMPLETED' && JSON.stringify(response.jobExecutions[0].progress) == '{"exported":1,"failed":0,"total":1}'
```

Comment:
Please make that check order independent.

Reply:
Made the code change as per the review comment.